akka-http-core/src/main/scala/akka/http/scaladsl/model/headers/headers.scala

```diff
@@ -57,9 +60,6 @@ sealed trait ModeledHeader extends HttpHeader with Serializable {
  * as they allow the custom header to be matched from [[RawHeader]] and vice-versa.
  */
 abstract class CustomHeader extends jm.headers.CustomHeader {
-  /** Override to return true if this header shouldn't be rendered */
-  def suppressRendering: Boolean = false
-
```

Comment:
Yes and yes.
I think it's good to force people to clearly define what their custom header is all about.
Adding a custom header shouldn't be an all too common thing. Deciding on whether and where you want it to be rendered should be good, also for documentation in your code.